README.md

```diff
@@ -376,18 +376,27 @@ This repository contains a curated list of awesome open source libraries that wi
 * [CuML](https://github.com/rapidsai/cuml) ![](https://img.shields.io/github/stars/rapidsai/cuml.svg?style=social) - cuML is a suite of libraries that implement machine learning algorithms and mathematical primitives functions that share compatible APIs with other RAPIDS projects.
 * [CuPy](https://github.com/cupy/cupy) ![](https://img.shields.io/github/stars/cupy/cupy.svg?style=social) - An implementation of NumPy-compatible multi-dimensional array on CUDA. CuPy consists of the core multi-dimensional array class, cupy.ndarray, and many functions on it.
 * [DeepSpeed MII](https://github.com/microsoft/DeepSpeed-MII) ![](https://img.shields.io/github/stars/microsoft/DeepSpeed-MII.svg?style=social) - DeepSpeed MII makes low-latency and high-throughput inference possible, powered by DeepSpeed.
+* [fastai](https://github.com/fastai/fastai) ![](https://img.shields.io/github/stars/fastai/fastai.svg?style=social) - fastai is a deep learning library which provides practitioners with high-level components that can quickly and easily provide state-of-the-art results in standard deep learning domains, and provides researchers with low-level components that can be mixed and matched to build new approaches.
```

Comment:
Fastai would not be computation, if anything would be deployment - however it woudl not make it in this list as too general, having said that, any frameworks built on top of fastapi would be welcome in the deployment and orchestration section (like MLServer)